### ISSUE TITLE
String literals wrongly highlighted

### ISSUE THREAD

**Issue by guijcl** [freest-lang/freest-language#1]
String literals are created using double quotes and are distinctively highlighted in the syntax highlighter of every minimally known programming language.

Some examples:
```Java
String s = "Test String";
```

```C
char s[] = "Test String";
```

```Rust
let s = "Test String";
```

```Haskell
let s = "Test String"
```

FreeST's syntax highlighter current version is missing this detail, ignoring the case of string literals which remain unhighlighted or wrongly highlighted. 
For example, writing a "let" in double quotes presents the "let" word as pink because it is a keyword. Also, the quotation character always remains white/unhighlighted. Please take a look at the demonstration in the image below.

![image](https://user-images.githubusercontent.com/54117581/233854437-6a19ef0c-c3ee-4c6e-855b-885376e05dfb.png)

As the showcase examples above show, string literals are always highlighted as a unique colour (including the quotation characters).

**Comment by diogo-mustachio-barros** [comment]
Can you check what version you are running? I've tested in Ubuntu and Windows 11 and can't reproduce the error with the latest version from the Marketplace

**Comment by guijcl** [comment]
I am using the 1.0.4 version of the extension available in the marketplace. I tried reinstalling the extension to update to a newer version but got the same results.

I am using MacOS.

**Comment by diogo-mustachio-barros** [comment]
There was in fact a bug introduced since commit [8ff6d6c](https://github.com/freest-lang/freest-language/commit/8ff6d6c9478100af3136fd8f0879e91ba5b2d4bd).

It is now fixed in commit [3924986](https://github.com/freest-lang/freest-language/commit/3924986f99b211539eae23c4a503409afc6fc1cd) (version 1.0.5).
